Generic-SQL adding oracle test playbook

ohaim1008 opened the pull request:
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26497

## Description
The generic-sql test playbook is failing sometimes, adding a simpler version of it for Oracle db.
 
## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

bakatzir commented on Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml:
```diff
@@ -0,0 +1,362 @@
+id: 1725220c-4672-4994-8489-3c3abd855aa4
+version: 2
```
id == name
version -1

bakatzir commented on Tests/conf.json:
```diff
@@ -639,7 +639,7 @@
         {
             "integrations": "Generic SQL",
-            "playbookID": "generic-sql",
+            "playbookID": "generic-sql-oracle",
```
what will happen to `generic-sql` do you plan to stop running it?

ohaim1008 replied:
it will run for mysql, one version of mssql (there is another test playbook that run with the other one) and postgesql.

ohaim1008 commented:
@bakatzir doing it here - https://github.com/demisto/content/pull/7999/

ohaim1008 commented:
@glicht Can I have your opinion?

We were having some issues with the oracle test playbook, from time to time it didn't delete a row in the table so I changed it to a simpler test playbook.
Locally (on self-server) I had no issues with either of the test playbooks but on the circle I got:
<img width="514" alt="Screen Shot 2020-07-15 at 12 58 37" src="https://user-images.githubusercontent.com/60567860/87531907-0f786380-c69b-11ea-84dd-0cefc3f8968b.png">
I added max_identifier_length=128 to create_engine (I'll change it so it will be added only in case Oracle db is used)

Currently the tests for 5.0 is failing due to an issue that was solved before (https://github.com/demisto/etc/issues/24218), the 'Test' is failing with the encrypted mssql instance, it works fine in 5.5 and 6.0 so I am wondering what could be the problem.

glicht commented:
> max_identifier_length

Don't change this setting as it isn't correct for older versions. The right thing to do is to disable warnings. I added a commit to deal with the warnings.

glicht commented:
> Currently the tests for 5.0 is failing due to an issue that was solved before (demisto/etc#24218), the 'Test' is failing with the encrypted mssql instance, it works fine in 5.5 and 6.0 so I am wondering what could be the problem.

I don't think it is failing because of the issue mentioned as the playbook test passes. It is failing after the instance upgrade. I think it was just a temporary network issue.

glicht commented on Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml:
```diff
@@ -0,0 +1,362 @@
+id: generic-sql-oracle
+version: -1
+contentitemexportablefields:
```
I think running demisto-sdk format will remove this section and other un-needed stuff.

ohaim1008 commented:
Failing in 5.0 - tested on self server.

Config a generic sql instance according to the `Microsoft SQL instance using encrypted connection` in `cof.json`.
Test - succeed.
Running `generic-sql-mssql-encrypted-connection` with the same instance- succeed.
No errors or warning were found.

@glicht - please approve force merge.

glicht commented on Packs/GenericSQL/ReleaseNotes/1_0_4.md:
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Generic SQL
+- Internal code improvements.
```
```suggestion
- Disabled Oracle driver warnings.
```

amshamah419 commented on Packs/GenericSQL/ReleaseNotes/1_0_4.md:
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Generic SQL
+- Disabled Oracle driver warnings.
```
```suggestion
- Disabled the warnings from the Oracle driver.
```

bakatzir commented:
Force merging on the 5.0 failure, let's track master and nightly @ohaim1008
